angular_token_auth.js

```diff
@@ -12,6 +12,7 @@
         LOGOUT_REDIRECT_URL: '/logout/',
         AUTH_HEADER_PREFIX: 'Token',
         ALLOWED_HOSTS: [],
+        STORAGE_METHOD: 'auto',
```

Comment:
Yes, what @henrahmagix said. I could leave the option out completely or set it to `null`, but it seemed more of a clue to leave it there because:
* It shows the option is available
* It shows the default behaviour, even if the value is not actually important